deployment/hasura/migrations/AerieMerlin/16_associate_external_datasets_with_simulation/up.sql

```diff
@@ -0,0 +1,16 @@
+alter table plan_dataset
+  add column associated_simulation_dataset_id integer default null,
+  add constraint simulation_dataset_references_simulation_dataset
```

Comment:
Proposing this name change as it's both clearer and shorter
```suggestion
  add constraint associated_sim_dataset_exists
```

Reply:
good call 👍